app/assets/stylesheets/repositories.scss

```diff
@@ -2,6 +2,7 @@
   height: auto;
   max-height: 400px;
   overflow-x: hidden;
+  text-transform: initial;
```

Comment:
Težka :banana: 
Could we do more nicer here or is this dropdown text transformed with css?